angularjs/angular.d.ts

```diff
@@ -1033,10 +1036,10 @@ declare module angular {
     interface IPromise<T> {
         /**
          * Regardless of when the promise was or will be resolved or rejected, then calls one of the success or error callbacks asynchronously as soon as the result is available. The callbacks are called with a single argument: the result or rejection reason. Additionally, the notify callback may be called zero or more times to provide a progress indication, before the promise is resolved or rejected.
-         *
+         * The successCallBack may return IPromise<void> for when a $q.reject() needs to be returned
```

Comment:
Improper grammar. It doesn't make sense to say a function parameter "can return" a certain type; rather it _is_ that type.

This should be "The successCallback __can be__ an IPromise in the case where $q.reject() needs to be returned."

Also, please include a trailing period `.`